docs/md_book/src/protocols.md

```diff
@@ -1,36 +1,41 @@
 # Networking protocols
 
-So first and possibly the important one is which protocol to use and when. Let’s first take a look at TCP and UDP.
+When building any networked application the first and possibly the important decision to make is which protocol to use and when. Laminar is built on top of UDP. Let’s first take a quick look at both TCP and UDP, then we'll explain why Laminar uses UDP.
```

Comment:
This is the only semantic change I made to this file. The main reason for the change was I wanted to explicitly state that Laminar is built using UDP, because I wanted to mention that in the "Connections" section.

Everything below this is my markdown autoformatting changes. I'm happy to revert those changes below if they're undesirable.